src/osp/universe/universe.h

```diff
@@ -124,18 +88,19 @@ struct CoSpaceTransform
     // Ignore and use m_parentSat's position and rotation instead if non-null
     Quaterniond     m_rotation;
     Vector3g        m_position;
+
     int             m_precision{10}; // 1 meter = 2^m_precision
 };
 
-
 struct CoSpaceSatData
 {
     uint32_t        m_satCount;
```

Comment:
Variable 'osp::universe::CoSpaceSatData::m_satCount' is uninitialized. Always initialize a member variable (type.6).

Warning
Variable 'osp::universe::CoSpaceSatData::m_satCount' is uninitialized. Always initialize a member variable (type.6).